ci-operator/step-registry/rosa/tests/rosa-tests-commands.sh

```
@@ -62,10 +62,22 @@ FOCUS=""
+tree = ET.parse(sys.argv[1])
+for elem in tree.iter():
+    for child in list(elem):
+        if child.tag in ('system-out', 'system-err'):
+            elem.remove(child)
+tree.write(sys.argv[1], xml_declaration=True, encoding='unicode')
+" "${JUNIT_XML}"
+fi
+cp "${JUNIT_XML}" "${ARTIFACT_DIR}/"
 
 log "Testing is finished and uploaded."
+exit ${test_exit}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)